pkgs/development/tools/buf/default.nix

```diff
@@ -7,16 +7,16 @@
 
 buildGoModule rec {
   pname = "buf";
-  version = "0.46.0";
+  version = "0.48.2";
 
   src = fetchFromGitHub {
     owner = "bufbuild";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-5mjk31HuPNO/RhmMhIm3dAZAED/Kk33ObjC8KbPKRxk=";
+    sha256 = "sha256-+l7CDqOq2S48tJqcmy60Yi6KaFnj1foQwA2rM2ISfTU=";
     leaveDotGit = true; # Required by TestWorkspaceGit
```

Comment:
Can we remove this and create a new git repo before running the tests?

Reply:
Yeah im just going to patch out the one test that needs it

Reply:
I did try just git init just in case but it also requires a commit so yeah patched it out. Git is still a check input though as it is used for another test with no issues